operations/wallet-service/jwtInterceptor.bal

```diff
@@ -14,7 +14,7 @@ const EMAIL = "email";
 service class JwtInterceptor {
     *http:RequestInterceptor;
     isolated resource function 'default [string... path](http:RequestContext ctx, http:Request req)
-    returns http:NextService|http:NotFound|http:Unauthorized|http:Forbidden|error? {
+    returns http:NextService|http:NotFound|http:Forbidden|error? {
```

Comment:
You can remove http:NotFound as well

Reply:
Yes, Thanks
Fixed [25abbb2](https://github.com/wso2/digiops-currency/pull/37/commits/25abbb2e796e3b8ce24b85c5193cab2c09a2ba9d)